handlers/line/line.go

```diff
@@ -236,12 +239,17 @@ func (h *handler) SendMsg(ctx context.Context, msg courier.Msg) (courier.MsgStat
 		batchCount++
 
 		if batchCount == maxMsgSend || (i == len(jsonMsgs)-1) {
-			if req, err := buildSendMsgRequest(authToken, msg.URN().Path(), batch); err == nil {
+			if req, err := buildSendMsgRequest(authToken, msg.URN().Path(), msg.ResponseToExternalID(), batch); err == nil {
 				rr, err := utils.MakeHTTPRequest(req)
 				log := courier.NewChannelLogFromRR("Message Sent", msg.Channel(), msg.ID(), rr).WithError("Message Send Error", err)
 				status.AddLog(log)
 
 				if err != nil {
+					msg, err := jsonparser.GetString(rr.Body, "message")
+					// don't retry messages for invalid reply token
```

Comment:
Because the `replyToken` must be of the last user message.

Reply:
Are you saying that if a user sends two messages "one" and "two".. if we use the reply token from "one" in a reply it will fail? Seems like that's a real possibility.

Should we back down to a push in this case?

Reply:
Exactly. I guess that way the users will have a greater sense of security regarding the functioning of the LINE channel, they will be aware that they will be consuming their plan limits (up to 500 for free plan) in triggering messages.

Reply:
So do you agree we should be switching to a push there or not?

Reply:
Flows for social networks are very much based on user's response, right? So if an "old" message fails and a retry consumes some limit, I think we should not switch to `push '.

Reply:
But it might still be interactive. If a contact is in a flow and responds to a question with two answer right after each other "yes", "of course", then what you are saying is the next flow question/response would fail completely because it would have a "response_to" id for the "yes" message. That seems wrong to me. Even if used sparingly it feels more right to try for a reply and if that doesn't work then use a push.

Reply:
Hmm, well noted!!

Reply:
@nicpottier FYI the last requested changes were made.